scripts/apps/monitoring/controllers/AggregateCtrl.ts

```diff
@@ -372,8 +372,8 @@ export function AggregateCtrl($scope, desks, workspaces, preferencesService, sto
         if (filterType === 'contentProfile') {
             if (!this.activeFilters.contentProfile.includes(filterValue._id)) {
                 this.activeFilters.contentProfile.push(filterValue._id);
-                const tag = {'key': filterValue._id, 'label': filterValue.label};
-                const type = 'Content profile';
+                const tag = {'key': filterValue._id, 'label': gettext(filterValue.label)};
```

Comment:
Only static strings can be passed to `gettext`. Don't attempt translating `filterValue.label` here. Instead, translate it at the point where it's a static string, so when it comes to this variable it's already translated.